endtoendtests/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E/HttpEndToEndTests.cs

```diff
@@ -42,22 +42,9 @@ public async Task HttpTrigger_ReturnsCustomCode()
         [Fact]
         public async Task HttpTriggerJavaClassLoader()
         {
-            // The e2e project has newer jars than the one we use in the worker.
-            // The purpose of this test will be called for three scenarios:
-            // 1. Java 11 -- Client code takes presence. -- works fine.
-            // 2. Java 8 with no Application settings, worker lib jars takes presence -- throw exception
-            // 3. Java 8 with with Application settings, worker lib jars takes presence -- works fine.
-
-            String value = Environment.GetEnvironmentVariable("FUNCTIONS_WORKER_JAVA_LOAD_APP_LIBS");
-            String java_home = Environment.GetEnvironmentVariable("JAVA_HOME");
-            if (java_home.Contains("zulu11") || (value != null && value.ToLower().Equals("true")))
-            {
-                Assert.True(await Utilities.InvokeHttpTrigger("HttpTriggerJavaClassLoader", "?&name=Test", HttpStatusCode.OK, "Test"));
-            }
-            else
-            {
-                Assert.True(await Utilities.InvokeHttpTrigger("HttpTriggerJavaClassLoader", "?&name=Test", HttpStatusCode.InternalServerError, ""));
-            }
+
+            Assert.True(await Utilities.InvokeHttpTrigger("HttpTriggerJavaClassLoader", "?&name=Test", HttpStatusCode.OK, "Test"));
```

Comment:
@TsuyoshiUshio I change the test cases here, before for java 8 if work lib is not found it will throw exception, but after the change this case should also works fine.

Reply:
Hi @kaibocai 
Looks good and thank you for noticing a lot of things and remove some constants. 
Ready for merge for v4 branch. After Ahmed is coming back, Let's test with integration testing.